### DISCUSSION TITLE
Freeplane 1.10.3 released

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#468] [Announcements]
It should make the recent UI changes available to most users.

**Comment by Tom582** [comment]
Thanks for the new update, Dimitry.

I just installed it successfully both under Linux Mint 19.3 and Windows 10. After playing a little bit with the new revision, here is what I noticed (so far).

- The two-row toolbar ribbon seems to be new. It allows to bundle the different tool icons in a more compact way than in the traditional toolbar, which is nice. However, it will take some time to get used to it.
- The default hot key to insert a new node has changed from "Ins" (old) to "Tab" (new).
- Alt key codes do work now under Windows 10 :+1: .
- Some of the icons in the menus are a bit small (e.g. see the the checkbox icons in the screenshot below taken under Linux). Maybe you want to increase their sizes in a future release.

![Freeplane 1-10-3 Menu Icons](https://user-images.githubusercontent.com/45515358/176003754-1cc397fa-0a09-4b5b-8058-3b2059731f0b.png)

Regards
Tom

**Comment by ldmpub** [comment]
Hi all,

Update done. Thanks 👍.

Is there a way to add (back) the "close button" (to close current MinMap) in the two-row toolbar ribbon? (it has vanished in version 1.10.3 and I missed it ... or maybe I need to change my glasses 😋? ... someone can just point me to it? ...).

More global question: how to customize the icons / buttons to fit personal needs? Is it already possible, is it planned?

**Reply by dpolivaev** [reply]
Which button? Could you post a screenshot?

**Reply by ldmpub** [reply]
Sorry I can not add screen shot (because previous Freeplane version is not installed anymore on my computer 😁).

Here is an extract of the "File Menu" where the command "Close" I miss is available: 
![image](https://user-images.githubusercontent.com/1452146/176025668-ed1274db-90c2-4812-a12a-dad9f23db6d9.png)

PS: Thanks for the very quick reply + I'm used to open a map for "instant" check / reading and then quickly close it (I'm using "close" button in conjunction with "Recent maps" command in "File" menu ...).
